common/api-review/database.api.md

```diff
@@ -7,7 +7,7 @@
 import { EmulatorMockTokenOptions } from '@firebase/util';
 import { FirebaseApp } from '@firebase/app';
 
-// @public
+// @public (undocumented)

```

Comment:
I don't think I see any change that would cause this, did this md not get checked in after a previous change?

Reply:
Fixed. The comment was accidentally removed by eslint.